cpp/tests/test_shuffler.cpp

```diff
@@ -484,22 +487,13 @@ class ShuffleInsertGroupedTest
     }
 
     void TearDown() override {
-        // resume progress thread - this will guarantee that shuffler progress
-        // function is marked as done. This is important to ensure that the test does
-        // not hang.
-        progress_thread->resume();
-
-        if (shuffler) {
-            shuffler->shutdown();
-        }
-
-        progress_thread->stop();
+        progress_thread = nullptr;
```

Comment:
I think the `progress_thread` will not stop if `shuffler` isn't stopped, will it?

Reply:
Yes, shuffler is stopped at the end of the test run (inside the test case)